examples/customresources/models/mygizmo/mygizmo.go

```diff
@@ -15,6 +15,15 @@ import (
 // Model is the full model definition.
 var Model = resource.NewModel("acme", "demo", "mygizmo")
 
+// Config is the gizmo model's config.
+type Config struct {
+	Arg string `json:"arg1"`
```

Comment:
Need comments for the json tags here, or else people might not understand why "Arg" here and "arg1" in the config.

Reply:
should I add a simple validate here? the original function would just panic I think

Reply:
Yeah, if there's a danger of a panic, that could be good to expand the example.

Reply:
Yeah, none of the attribute map functions return errors:
```go
// String attempts to return a string present in the map with
// the given name; returns an empty string otherwise.
func (am AttributeMap) String(name string) string {
	if am == nil {
		return ""
	}
	x := am[name]
	if x == nil {
		return ""
	}

	if s, ok := x.(string); ok {
		return s
	}

	panic(errors.Errorf("wanted a string for (%s) but got (%v) %T", name, x, x))
}
```

So the behaviors of this model without a string input to arg! would be to panic continuously. I'll add a Validate.